app/javascript/src/components/editor/Editor.svelte

```diff
@@ -28,8 +29,14 @@
   let defaults = {}
   let loading = true
 
-  $: if ($currentProject && $sortedItems?.length && $currentItem && !Object.keys($currentItem).length)
-    $currentItem = $sortedItems.filter(i => i.type == "item")?.[0] || {}
+  let isCurrentItemInherentlyHidden = false
+
+  $: if ($currentProject && $sortedItems?.length) {
+    if ($currentItem && !Object.keys($currentItem).length)
+      $currentItem = $sortedItems.filter(i => i.type == "item")?.[0] || {}
+
+    isCurrentItemInherentlyHidden = isInherentlyHidden($currentItem)
+  }
```

Comment:
I would prefer to see these as separate statements to keep the logic separate, would also prevent you from having to declare the variable separately

```suggestion
  $: if ($currentProject && $sortedItems?.length && $currentItem && !Object.keys($currentItem).length)
  let isCurrentItemInherentlyHidden = false
    $currentItem = $sortedItems.filter(i => i.type == "item")?.[0] || {}

  $: isCurrentItemInherentlyHidden = $currentItem && isInherentlyHidden($currentItem)
```

Reply:
That looks a little weird, but I assume you meant this
```suggestion
  $: if ($currentProject && $sortedItems?.length && $currentItem && !Object.keys($currentItem).length)
    $currentItem = $sortedItems.filter(i => i.type == "item")?.[0] || {}

  let isCurrentItemInherentlyHidden = false
  $: isCurrentItemInherentlyHidden = $currentItem && isInherentlyHidden($currentItem)
```

Reply:
No, you don't need to declare the variable separately